scripts/lib/CIME/case_submit.py

```diff
@@ -42,7 +42,8 @@ def _submit(case, job=None, resubmit=False, no_batch=False, skip_pnl=False, batc
     # flag will stay in effect for the duration of the RESUBMITs
     env_batch = case.get_env("batch")
     if not resubmit:
-        case.set_value("IS_FIRST_RUN", True)
+        if case.get_value("TEST"):
+            case.set_value("IS_FIRST_RUN", True)
```

Comment:
Thanks for noticing, I don't think that second case would have been caught by any of our testing.